# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

geometry_msgs/Pose2D pose
string patch_type
int32 ID
string direction